jax/interpreters/batching.py

```diff
@@ -444,9 +441,10 @@ def batch_jaxpr(jaxpr, size, batched, instantiate):
     return jaxpr_out, batched_out()
 
 
-# It is assumed that all collectives specified below are commutative
-# and associative over axis names if they support tuples. That is,
-# they have to satisfy:
-#   collective(x, ('i', 'j')) == collective(x, ('j', 'i'))
-#                             == collective(collective(x, 'j'), 'i')
+# Those rules can assume that the collective is only carried out throughout
```

Comment:
Is this comment ("Those rules can assume...") still meant to be here? If so it seems unclear. Maybe it was meant to move with the rest of the comment to lax_parallel.py?